pkg/controller/awsmachineclass_test.go

```diff
@@ -121,21 +122,21 @@ var _ = Describe("machineclass", func() {
 				waitForCacheSync(stop, controller)
 
 				action := data.action
-				machineClass, err := controller.controlMachineClient.AWSMachineClasses(TestNamespace).Get(action.machineClassName, metav1.GetOptions{})
+				machineClass, err := controller.controlMachineClient.AWSMachineClasses(TestNamespace).Get(context.TODO(), action.machineClassName, metav1.GetOptions{})
```

Comment:
I would think we need to pass a bg.context here in this file as well.

Reply:
Most of the tests have `context.TODO(`) set. So, should be changed across all tests to `context.Background()`?

Reply:
Sorry, my bad. I mistook it to be for `pkg/controller/awsmachineclass`. The test files can continue to have context.TODO() I think. 

Ignore my previous comment. The page loading was too slow, hence got confused I think.